Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityGesturesProfileInspector.cs

```diff
@@ -53,18 +56,27 @@ protected override void OnEnable()
             thisProfile = target as MixedRealityGesturesProfile;
             Debug.Assert(thisProfile != null);
 
-            if (MixedRealityToolkit.Instance.ActiveProfile.IsInputSystemEnabled &&
-                MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.InputActionsProfile != null)
+            UpdateGestureLabels();
+
+            if (!MixedRealityToolkit.IsInitialized ||
+                !MixedRealityToolkit.Instance.ActiveProfile.IsInputSystemEnabled ||
+                MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.InputActionsProfile != null ||
```

Comment:
>! [](start = 98, length = 1)

I think this should be == (because it's supposed to be the opposite of what it used to be.

I was noticing a ton of exceptions in the unity editor when scrolling up and down the input profile I think due to this.